gravitee-apim-console-webui/src/management/application/details/user-group-access/members/application-general-members.component.ts

```diff
@@ -161,32 +163,36 @@ export class ApplicationGeneralMembersComponent {
       .subscribe();
   }
 
-  public removeMember(member: any) {
-    this.matDialog
-      .open<GioConfirmDialogComponent, GioConfirmDialogData>(GioConfirmDialogComponent, {
-        width: '500px',
-        data: {
-          title: `Remove Application member`,
-          content: `Are you sure you want to remove "<b>${member.displayName}</b>" from this Application members? <br>This action cannot be undone!`,
-          confirmButton: 'Remove',
-        },
-        role: 'alertdialog',
-        id: 'confirmMemberDeleteDialog',
-      })
-      .afterClosed()
-      .pipe(
-        filter((confirm) => confirm === true),
-        switchMap(() => this.applicationMembersService.delete(this.activatedRoute.snapshot.params.applicationId, member.id)),
-        tap(() => {
-          this.snackBarService.success(`"${member.displayName}" has been deleted`);
-        }),
-        catchError(({ error }) => {
-          this.snackBarService.error(error.message);
-          return EMPTY;
-        }),
-        takeUntil(this.unsubscribe$),
-      )
-      .subscribe(() => this.ngOnInit());
+  public removeMember(member: MemberDataSource) {
+    if (member.notSaved) {
+      this.onReset();
```

Comment:
hum this is very strange no ? do you expect to reset the whole form when you delete an item from the list? 
what about juste find member here into `this.membersTable` and remove it ?